### PR TITLE
[R4R] re-enable claiming rewards validator vesting accounts

### PR DESCRIPTION
**Pull request by karzak**

### CODE REVIEW

File: x/incentive/client/cli/tx.go [hunk 1]
```diff
@@ -63,9 +65,79 @@ func getCmdClaimCdp(cdc *codec.Codec) *cobra.Command {
...
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func getCmdClaimHardVVesting(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "claim-hard-vesting [multiplier]",
```
Comment (rhuairahrighairidh) [inline x/incentive/client/cli/tx.go:140]: Is this meant to be `claim-hard`? Currently it has the same name as the command for claiming hard rewards for a vv account.

**Review by rhuairahrighairidh — APPROVED** [review]
haven't tested locally, but looks good
Left a couple of questions

File: x/incentive/types/codec.go [hunk 2]
```diff
@@ -21,4 +21,6 @@ func RegisterCodec(cdc *codec.Codec) {
 	// Register msgs
 	cdc.RegisterConcrete(MsgClaimUSDXMintingReward{}, "incentive/MsgClaimUSDXMintingReward", nil)
 	cdc.RegisterConcrete(MsgClaimHardReward{}, "incentive/MsgClaimHardReward", nil)
+	cdc.RegisterConcrete(MsgClaimUSDXMintingRewardVVesting{}, "incentive/MsgClaimUSDXMintingRewardVVesting", nil)
```
[the remark this answers is not in the document]
Comment (karzak) [reply x/incentive/types/codec.go:24]: Good catch, limit is 39 chars

File: x/incentive/client/cli/tx.go (hunk 1 above)
Comment (karzak) [reply x/incentive/client/cli/tx.go:140]: Yep, reversed the function names.

**Pull request merged**